src/native-stack/types.tsx

```diff
@@ -286,6 +287,14 @@ export type NativeStackNavigationOptions = {
    * @platform android
    */
   navigationBarHidden?: boolean;
+  /**
```

Comment:
Supported values would be nice here too.

Reply:
[bcfbc09](https://github.com/software-mansion/react-native-screens/pull/1649/commits/bcfbc093d1237092400b6e715c3967e5d11d6e08)